packages/dd-trace/test/instrumenter.spec.js

```diff
@@ -416,6 +416,18 @@ describe('Instrumenter', () => {
     })
   })
 
+  describe('with service mapping', () => {
+    it('should change the service name in plugin config', () => {
```

Comment:
Since this didn't work in the original version of this PR, should there be a test for using `DD_SERVICE_MAPPING` with `tracer.use()`?